src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsParserStateObject.cs

```diff
@@ -13,6 +13,11 @@
 
 namespace Microsoft.Data.SqlClient
 {
+#if NETFRAMEWORK
+    using PacketHandle = IntPtr;
```

Comment:
An alternative would be to add a struct wrapping IntPtr instead of an alias, but would require refactoring a few more lines of code.

The advantage of using a struct is that the alias has to be defined in every file it is used, rather than once.